src/triage/component/catwalk/evaluation.py

```diff
@@ -214,7 +227,14 @@ def evaluate(
         labels_sorted = numpy.array(labels_sorted)
 
         evaluations = []
-        for group in self.metric_groups:
+        if matrix_type.lower() == "train":
```

Comment:
I am not crazy about these magic strings. 

My first thought: make this into a defined enum
My second thought: since we have other stuff than strings hanging off this (table names, for instance), instead of an enum just make it a class that encapsulates all this, in "Tell, Don't Ask" fashion (https://martinfowler.com/bliki/TellDontAsk.html)

```
class MatrixType(object):
    pass

class TrainMatrixType(object):
    string_name = 'train'
    table_obj = TrainEvaluation
```

Then this method just gets a MatrixType object passed to it instead of a string.